src/components/TransactionMenu/index.tsx

```diff
@@ -92,10 +182,18 @@ export default function TransactionMenu({
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
-        <ClearTransactionsBtn />
         <ZeroTransactions />
         {net.transactions.map((t) => TransactionMenuItem(t))}
+        <AllTransactionsBtn />
+        <ClearTransactionsBtn />
```

Comment:
whats ClearTransactions for?

Reply:
It clears the array of all submitted transactions, similar to the delete button it is not required just added it as an ux feature.

Reply:
hm i'm not sure we should have any clear/delete functionality at all - don't think it necessarily provides user value, and could also be confusing in terms of what it means after refresh/coming back to app.